core/query/annotated.go

```diff
@@ -34,7 +34,7 @@ type AnnotatedInput struct {
 	Amount          uint64             `json:"amount"`
 	IssuanceProgram chainjson.HexBytes `json:"issuance_program,omitempty"`
 	ControlProgram  chainjson.HexBytes `json:"control_program,omitempty"`
-	SpentOutput     *SpentOutput       `json:"spent_output,omitempty"`
```

Comment:
Per conversation with @oleganza, we don't have conviction that the old `spent_output` object can or should be removed from a product perspective. In the interest of moving ahead with a large PR, we'll leave a comment here and file the lack of true annotation as a bug.